include/rpm/rpmts.h

```diff
@@ -37,7 +37,9 @@ enum rpmtransFlags_e {
     RPMTRANS_FLAG_NOCONTEXTS	= (1 <<  8),	/*!< from --nocontexts */
     RPMTRANS_FLAG_NOCAPS	= (1 <<  9),	/*!< from --nocaps */
     RPMTRANS_FLAG_NODB		= (1 << 10),	/*!< from --nodb */
-    /* bits 11-15 unused */
+    /* bits 11-13 unused */
+    RPMTRANS_FLAG_NOPREUNTRANS	= (1 << 14),	/*!< from --preuntrans */
```

Comment:
Oh. I did multiple versions of this patch, some with scriptlet specific disablers and some without, and the final version was supposed to be *without*, ie there are no --nopreuntrans cli switches either, so these RPMTRANS_FLAG_* additions shouldn't be there at all.

Reply:
Oh, actually this *was* mentioned in the commit message:

"Cheapskate on transaction flags are reuse pre/posttrans flags for these uninstall counterparts 
too, adding separate flags and disablers just doesn't seem worth it, especially as we're quite short of free bits." 

Always a good idea to explicitly explain things in the commit message, that way you have a double bookkeeping: if the two disagree, something isn't right :laughing: 
Thanks for spotting!

Reply:
Oh, of course. I swear I read the commit message, but somehow skimmed through it too quickly without realizing what's written in it, apparently :smile:

Reply:
> "Cheapskate on transaction flags are reuse pre/posttrans flags [...]

Ah, at the risk of sounding nitpicky, wasn't the "are" supposed to be "and"? :sweat_smile: Otherwise the sentence sounds non-sensical (to me) :smile:

Reply:
Yep, another typo, only now in the commit message :facepalm: Noticed that myself as well, now fixed.